src/helpers/pagination.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
I think this task was fulfilled by @Onyimatics , please make amends. 

see this article on how to rebase in git:
https://blog.algolia.com/master-git-rebase/